sensor_msgs/PointCloud cloud
---
tabletop_msgs/TableTopObject object
string model_name
float32 score
